lib/utilities/paginator.rb

```diff
@@ -0,0 +1,106 @@
+module Utilities
+  module Paginator
+    def items_to_display(model:, items_per_page: 5, current_page: 1)
+      @items_per_page = items_per_page
+      @current_page = current_page.to_i
```

Comment:
Yeah I'd like to see that done in the controller. To have current page referring to a string or integer depending on where it is being used is odd. Also is there an instance we need it to be a string? Usually the string "1" is pretty useless.